### DISCUSSION TITLE
Add snippets for Remix loaders and actions in VSCode

### DISCUSSION THREAD

**Discussion by hakimLyon** [epicweb-dev/epic-stack#554] [Ideas]
I suggest adding snippets in VSCode to ease development. These snippets will assist developers in quickly generating boilerplate code for loaders, actions, and other common parts of a Remix application.

## References:
[remix-snippets](https://www.jacobparis.com/content/remix-snippets)

**Comment by kentcdodds** [comment]
I think this is a good idea. I would prefer the built-in snippets match the built-in code style. I would welcome a PR to add them to the `.vscode` directory (potentially I could put together an extension for this in the future).

**Reply by hakimLyon** [reply]
Hi ! Snippets are now added to the .vscode directory. PR  #558.

**Reply by kentcdodds** [reply]
Thanks!
